packages/ui/src/component/UITransform.ts

```diff
@@ -0,0 +1,59 @@
+import { Entity, Transform, Vector2, deepClone, ignoreClone } from "@galacean/engine";
+
+export class UITransform extends Transform {
+  @deepClone
+  private _size: Vector2 = new Vector2(100, 100);
+  @deepClone
+  private _pivot: Vector2 = new Vector2(0.5, 0.5);
+
+  get size(): Vector2 {
```

Comment:
May be we need a Size Class, use size.x or size.y to set width or height not very suitable